packages/opentelemetry-node/src/index.ts

```diff
@@ -1,3 +1,4 @@
 import '@sentry/tracing';
 
 export { SentrySpanProcessor } from './spanprocessor';
+export { SentryPropogator } from './propogator';
```

Comment:
I think this should be `propagator`, not `propogator` ;) (both in the filename & class name)

Reply:
Yup, fixed this afterwards, see: https://github.com/getsentry/sentry-javascript/pull/6109

I literally spelled it wrong everywhere